### PR TITLE
test: add auto case "Azure Tree View"

### PR DESCRIPTION
**Pull request by QinghuiMeng-M**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -138,3 +138,4 @@ extensions
 
 # templates from widdershins
 templates/openapi3
+/.vs
```
Comment (wenytang-ms) [inline .gitignore:141]: what's this?

Comment (QinghuiMeng-M) [reply .gitignore]: sorry, it is my local change, removed it

File: src/test/e2e/tests/validateAzureTreeView.test.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    await workbox.waitForTimeout(Timeout.PREPARE_TEST);
+    // wait API Center extension installed on VS Code.
+    expect(await VscodeOperator.isSideTabItemExist(workbox, "API Center")).toBeTruthy();
+    await VscodeOperator.activeSideTab(workbox, VSCode.TAB_API_CENTER, Timeout.PREPARE_EXT);
+
+    //expand and validate tree items
+    await VscodeOperator.clickTreeItemChildLinkByText(workbox, "Teams Cloud - E2E Testing with TTL = 1 Days", "Teams Cloud - E2E Testing with TTL = 1 Days");
+    expect(await VscodeOperator.isTreeItemExist(workbox, "apicentertest001")).toBeTruthy();
+    await VscodeOperator.clickTreeItem(workbox, "apicentertest001");
+    expect(await VscodeOperator.isTreeItemExist(workbox, "APIs")).toBeTruthy();
```
Comment (wenytang-ms) [inline src/test/e2e/tests/validateAzureTreeView.test.ts:21]: it's better to check all the items here, not only APIs, but also Environments and Rules.

File: src/test/e2e/tests/validateAzureTreeView.test.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    await VscodeOperator.clickTreeItemChildLinkByText(workbox, "APIs", "APIs");
+    expect(await VscodeOperator.isTreeItemExist(workbox, "callback-example")).toBeTruthy();
+    await VscodeOperator.clickTreeItem(workbox, "callback-example");
+    expect(await VscodeOperator.isTreeItemExist(workbox, "Versions")).toBeTruthy();
```
Comment (wenytang-ms) [inline src/test/e2e/tests/validateAzureTreeView.test.ts:25]: same here, better to check Deployments exists.

File: src/test/e2e/tests/validateAzureTreeView.test.ts (hunk 2 above)
Comment (QinghuiMeng-M) [reply src/test/e2e/tests/validateAzureTreeView.test.ts:21]: got it, added

File: src/test/e2e/tests/validateAzureTreeView.test.ts (hunk 3 above)
Comment (QinghuiMeng-M) [reply src/test/e2e/tests/validateAzureTreeView.test.ts:28]: got it, added

**Review by wenytang-ms — APPROVED** [review]

**Review by formulahendry — APPROVED** [review]

**Pull request merged**